### PR TITLE
Add HMI Case HQ project to flux

### PR DESCRIPTION
**Pull request by chris-pateman**

### CODE REVIEW

File: k8s/environments/stg/common-overlay/flux-patch.yaml [hunk 1]
```diff
@@ -102,3 +102,13 @@ spec:
...
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: hmi-case-hq-emulator
+  namespace: hmi
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/hmi/case-hq-emulator:latest
```
Comment (timja) [inline k8s/environments/stg/common-overlay/flux-patch.yaml:114]:
```
$ docker pull sdshmctspublic.azurecr.io/hmi/case-hq-emulator
Using default tag: latest
Error response from daemon: manifest for sdshmctspublic.azurecr.io/hmi/case-hq-emulator:latest not found: manifest unknown: manifest tagged by "latest" is not found
```
🚫 blocker

File: k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml [hunk 2]
```diff
@@ -123,6 +123,12 @@ spec:
                     name: VH
                     recurse: true
                     title: Video Hearings Dashboard
+                - buildMonitor:
```
Comment (timja) [inline k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml:126]: looks unrelated to the change? I would separate to a separate PR normally

not a blocker

**Review by timja — CHANGES_REQUESTED** [review]

File: k8s/environments/stg/common-overlay/flux-patch.yaml (hunk 1 above)
Comment (chris-pateman) [reply k8s/environments/stg/common-overlay/flux-patch.yaml:114]: This is now pushing to acr, but PR is failing due to this missing config :)
https://github.com/hmcts/hmi-case-hq-emulator/pull/2

File: k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml (hunk 2 above)
Comment (chris-pateman) [reply k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml:126]: will take on board for next PRs
